src/main/java/in/muthukumari/servlet/CustomerDetailServlet.java

```diff
@@ -38,29 +43,41 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
 		String balanceAmount = request.getParameter("balanceAmount");
 		try {
-		long accNumberLong = Long.parseLong(accNumber);
-		long mobileNumberLong = Long.parseLong(mobileNumber);
-		int atmPinNumberInt = Integer.parseInt(atmPinNumber);
-		long atmNumberLong = Long.parseLong(atmNum);
-		double balanceAmonuDob = Double.parseDouble(balanceAmount);
-		customer.setBalanceAmount(balanceAmonuDob);
-		customer.setAccountNumber(accNumberLong);
-		customer.setAtmNumber(atmNumberLong);
-		customer.setAtmPinNumber(atmPinNumberInt);
-		customer.setMobileNumber(mobileNumberLong);
-		}
-		catch(Exception e) {
-			e.printStackTrace();
+			long accNumberLong = Long.parseLong(accNumber);
+			long mobileNumberLong = Long.parseLong(mobileNumber);
+			int atmPinNumberInt = Integer.parseInt(atmPinNumber);
+			long atmNumberLong = Long.parseLong(atmNum);
+			double balanceAmonuDob = Double.parseDouble(balanceAmount);
+			customer.setBalanceAmount(balanceAmonuDob);
+			customer.setAccountNumber(accNumberLong);
+			customer.setAtmNumber(atmNumberLong);
+			customer.setAtmPinNumber(atmPinNumberInt);
+			customer.setMobileNumber(mobileNumberLong);
+		} catch (Exception e) {
+			logger.info(e.getMessage());
 		}
 		// set customer bank details to the CustomerBankDetail class
 		customer.setUserName(name);
 		customer.setBankName(bankName);
 		customer.setBranchName(branchName);
 		customer.setIfscCode(ifsc);
-		
-		response.sendRedirect("DisplayCustomerBankDetail.jsp?" + "name=" + name + "&bankName=" + bankName
-				+ "&branchName=" + branchName + "&ifsc=" + ifsc + "&accNumber=" + accNumber + "&atmNum=" + atmNum
-				+ "&atmPinNum=" + atmPinNumber + "&mobileNum=" + mobileNumber + "&balanceAmount=" + balanceAmount);
+
+		try {
+			boolean isAdded = CustomerBankDetailService.addCustomerDetail(customer);
+			
+			if (isAdded) {
+				response.sendRedirect("DisplayCustomerBankDetail.jsp?name=" + name + "&bankName=" + bankName
+						+ "&branchName=" + branchName + "&ifsc=" + ifsc + "&accNumber=" + accNumber + "&atmNum="
+						+ atmNum + "&atmPinNum=" + atmPinNumber + "&mobileNum=" + mobileNumber + "&balanceAmount="
+						+ balanceAmount);
+			} else {
+				String errorMsg = "Invalid Data";
+				response.sendRedirect("CustomerBankDetail.jsp?errorMsg=" + errorMsg);
+			}
+		} catch (IOException | DBException | CustomerRepeatedException | ClassNotFoundException | SQLException e) {
```

Comment:
handle DB and service exception in service class

Reply:
Solve this issue